src/main/kotlin/calculator/Operator.kt

```diff
@@ -0,0 +1,38 @@
+package calculator
+
+import java.lang.IllegalArgumentException
+import java.util.function.BiFunction
+
+enum class Operator(
+    private val operator: String,
+    private val expression: BiFunction<Int, Int, Int>
```

Comment:
함수형 인터페이스 활용 👍👍👍🤩

BiFunction의 import 문을 보시면, 해당 인터페이스는 자바 코드를 import하고 있어요! 
해당 코드를 `코틀린스럽게` 아래와 같이 바꿀 수 있어요!! 🙂 (덤으로 자바의존성도 없앨 수 있어요 🕺🕺🕺)

```suggestion
    private val expression: (Int, Int) -> Int
```

Reply:
이렇게도 줄일 수 있네요~ 전보다 훨씬 코틀린스러운 방식인 것 같아요 해당 부분 반영해 볼게요!